src/plugins/console/public/styles/_app.scss

```diff
@@ -128,11 +137,15 @@
   min-width: $euiSizeL;
 }
 
-.conApp__resizer {
-  @include kbnResizer;
+.conApp__resizerButton {
   // Give the aria selection border priority when the divider is selected on IE11 and Chrome
   z-index: $euiZLevel1;
   background-color: $euiColorLightestShade;
+  // The margin ensures that the resizer doesn't cover the top border of the selected request
```

Comment:
@ElenaStoeva I am not seeing what this does exactly. Can you provide screenshots of what this addresses? I see the UI move as I toggle these styles on/off in the Inspector, but nothing appears to be 'covered up'. Thanks!

Reply:
Sure! When Console is in vertical layout, the resizer can cover the top border of the selected request in the output. Without the margin:
<img width="850" alt="Screenshot 2024-09-05 at 23 35 42" src="https://github.com/user-attachments/assets/1dd98b85-17fd-49bb-92df-be447f224b24">


With the margin:
<img width="856" alt="Screenshot 2024-09-05 at 23 32 10" src="https://github.com/user-attachments/assets/75fa1a12-da1e-4902-be58-0ce9332a3260">

Reply:
For the second margin, it makes the resizer not cover the input editor's ruler partially, when Console is in horizontal layout.

With the margin:
<img width="111" alt="Screenshot 2024-09-05 at 23 38 20" src="https://github.com/user-attachments/assets/139f7165-b2cd-428b-9579-73dd4ae58e8f">


Without:
<img width="111" alt="Screenshot 2024-09-05 at 23 38 03" src="https://github.com/user-attachments/assets/49559b6d-7ef2-4c18-9e6b-a656f4bee1a2">